docs/operating-scylla/security/authentication.rst

```diff
@@ -31,10 +31,10 @@ Procedure
 
        cqlsh -u cassandra -p cassandra
 
-   .. warning::
+   .. note::
 
-      Before proceeding  to the next step, we highly recommend creating a custom superuser 
-      to ensure security and prevent performance degradation.
+      Before proceeding  to the next step, we recommend creating a custom superuser
+      to improve security.
```

Comment:
Creating another superuser without deleting the old one first doesn't "improve", nor "ensure" security :-(

Wouldn't **changing** the password of the existing user, or deleting it, be the appropriate advice for having any level of security? (it's not even "improving" security - there is zero security if you keep the default password).

Reply:
yes there should be mention of disabling the old one, it's already in  create superuser procedure linked below

Reply:
I think we're falling deeper into the abyss :-(
First, https://opensource.docs.scylladb.com/stable/operating-scylla/security/create-superuser.html also includes the text about QUORUM which is (supposedly?) no longer true so perhaps should be removed. But in fact, the text there says that "the credentials for the default superuser cassandra are read with a consistency level of QUORUM" - not that the tables are read with QUORUM. And I still see logic like that in the code! So maybe it's still true? But does it matter? Shouldn't matter if credentials are cached, but maybe they aren't (as we saw in Alternator)?

If all that QUORUM business was gone, then for security all you need to do is to change the password of the existing "cassandra" role. I agree that this role also has a stupid name, so maybe it makes sense to replace it, but that is neither a security nor performance issue.

Reply:
credentials are never cached except in alternator where they are 😂

but you are right it is another miss, I have to remove the quorum mention, I didn't expect that this doc go so deep into the details...